games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -76,7 +76,7 @@ BUILD()
 {
 	mkdir -p build && cd build
 	cmake .. \
-		-DFG_DATA_DIR:STRING="/boot/system/data/flightgear" \
+		-DFG_DATA_DIR:STRING="`finddir B_SYSTEM_DATA_DIRECTORY` /flightgear" \
```

Comment:
Oops, there's an extra space:
```
-		-DFG_DATA_DIR:STRING="`finddir B_SYSTEM_DATA_DIRECTORY` /flightgear" \
+		-DFG_DATA_DIR:STRING="`finddir B_SYSTEM_DATA_DIRECTORY`/flightgear" \
```
(BTW, I just noticed I had eaten the back-quote in my comment.)